tigerpath/majors_and_certificates/degrees/BSE_2018.json

```diff
@@ -256,40 +241,36 @@
       "name": "Humanities and Social Sciences",
       "max_counted": 1,
       "min_needed": 2,
-      "description": null,
-      "explanation": null,
+      "explanation": "<b>Humanities and social science</b><br><i>7 total courses in 4 areas</i>",
       "pdfs_allowed": true,
       "req_list": [
         {
           "name": "Four Areas",
           "max_counted": 1,
           "min_needed": 4,
-          "description": "Humanities and social science courses in 4 areas",
-          "explanation": "The humanities and social science courses must include one course in four of the six areas listed below:\n\nEpistemology and Cognition (EC)\n\nEthical Thought and Moral Values (EM)\n\nForeign Language (at the 107/108 level or above)\n\nHistorical Analysis (HA)\n\nLiterature and the Arts (LA)\n\nSocial Analysis (SA)",
+          "explanation": "The humanities and social science courses must include one course in four of the six areas listed below:\n\n- Epistemology and Cognition (EC)\n- Ethical Thought and Moral Values (EM)\n- Foreign Language (at the 107/108 level or above)\n- Historical Analysis (HA)\n- Literature and the Arts (LA)\n- Social Analysis (SA)",
           "req_list": [
             {
               "name": "Epistemology and Cognition",
               "max_counted": 1,
               "min_needed": null,
-              "description": "Epistemology and Cognition (EC)",
-              "explanation": "The requirement in epistemology and cognition will introduce you to the critical study of the nature, sources and bounds of human knowledge. Courses in this category examine topics ranging from the basic perceptual capabilities that humans share with other animals to the distinctively human capacity for language; from theoretical models of human knowledge to empirical models of an individual's cognitive abilities; from the historical record of collective inquiry in the sciences and elsewhere to informed speculations on the outer limits of what is knowable.",
+              "explanation": "<b>Epistemology and Cognition (EC)</b><hr>The requirement in epistemology and cognition will introduce you to the critical study of the nature, sources and bounds of human knowledge. Courses in this category examine topics ranging from the basic perceptual capabilities that humans share with other animals to the distinctively human capacity for language; from theoretical models of human knowledge to empirical models of an individual's cognitive abilities; from the historical record of collective inquiry in the sciences and elsewhere to informed speculations on the outer limits of what is knowable.",
```

Comment:
Do we really need to repeat the title in the explanation? It's redundant, since the title is already in the popover header.

![Untitled](https://user-images.githubusercontent.com/4218237/87891302-598c7c80-c9ee-11ea-8854-a4f26d2eb30a.png)

Reply:
I honestly just wanted to include the two-letter code (EC), but it would look weird to just have that by itself.
Do you have an idea for how to do that?

Reply:
Chatted with Barak - we decided to include it in the explanation